### PR TITLE
Nominate developers in embulk-input-sqlserver

### PR DESCRIPTION
**Pull request by dmikurube**
Maven Central needs the `<developers>` section available in `pom.xml`. Nominating developers who have contributed in `embulk-input-sqlserver` from the Git log.

### CODE REVIEW

File: embulk-input-sqlserver/build.gradle [hunk 1]
```diff
@@ -11,3 +11,38 @@ embulkPlugin {
...
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Muga Nishizawa"
+                        email = "muga.nishizawa@gmail.com"
+                    }
+                    developer {
+                        name = "Shintaro Kimura"
+                        email = "kimura@enigmo.co.jp"
+                    }
```
Comment (dmikurube) [inline embulk-input-sqlserver/build.gradle:35]: @kmrshntr Hi!

We're going to upload `embulk-input-jdbc` (including `embulk-input-sqlserver`) to Maven Central, and Maven Central needs a `<developers>` section in `pom.xml`. On this occasion, we're trying to nominate past important contributors in the `<developers>` section.

We wonder if you're okay to nominate you there on Maven Central, from your work in #137 and #139. I picked up your name and email from the Git commit log, which are already public.

Please let me know if you don't want, or if you want another name/email.

Comment (kmrshntr) [reply embulk-input-sqlserver/build.gradle:35]: I am honored to be nominated. The name is OK. Please use my personal email address kmrshntr[at]gmail.com. 
Thank you.

Comment (dmikurube) [reply embulk-input-sqlserver/build.gradle]: Thanks! We're happy to have you there. Updated the email in 69525862759f2807784cfccfbf0b64b0f0e27bdc.

**Review by vietnguyen-td — APPROVED** [review]
LGTM

**Comment by hito4t** [comment]
@dmikurube 
Thank you!

**Review by hito4t — APPROVED** [review]

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**